content/dota_addons/legion_td/panorama/styles/custom_game/information_panel.css

```diff
@@ -0,0 +1,141 @@
+#InformationPanelRoot
+{
+	width: 1200px;
+	height: 776px;
+	padding: 0px;
+	overflow: clip;
+	flow-children: none;
+	border: 1px solid #5e6869;	
+	align: center center;
+	margin-bottom: 100px;
+	
+	background-color: #000;
+	background-image: url("s2r://panorama/images/backgrounds/gallery_background_png.vtex");
+	background-size:100% 100%;
+    box-shadow: fill #00000066 -4px -4px 8px 8px;
```

Comment:
keep one indention level (which should be the one I used before)

Reply:
I will stop marking this for the rest of the request but you should still change it. Thanks!